proto/spire/api/server/local_authority/v1/local_authority.proto

```diff
@@ -0,0 +1,174 @@
+syntax = "proto3";
+package spire.api.server.local_authority.v1;
+option go_package = "github.com/spiffe/spire-api-sdk/proto/spire/api/server/local_authority/v1;local_authorityv1";
+
+// The LocalAuthority service provides a way to manage the signing keys (and
+// related material) of the SPIRE Server exposing it.
+service LocalAuthority {
+        // GetJWTAuthorityState returns the state of all locally configured
+        // JWT authorities.
+        rpc GetJWTAuthorityState(GetJWTAuthorityStateRequest) returns (GetJWTAuthorityStateResponse);
```

Comment:
what is included in get authority state? I have the feeling we must not expose expired (since we are going to cleanup all expired authorities)
I'm not sure when "Prepared" authority can be useful, maybe to know if it is was exposed and want to prepare a new one?